test-scripts/run-presto-kerberos-integration-test.sh

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bash
+# runs one or more Metabase test(s) against a Kerberized Presto instance
+
+# install clojure version needed for Metabase
+curl -O https://download.clojure.org/install/linux-install-1.10.3.933.sh
+chmod +x linux-install-1.10.3.933.sh
+./linux-install-1.10.3.933.sh
+
+RESOURCES_DIR=/app/source/resources
+
+# ensure the expected files are in place, in the resources dir
+if [ ! -f "$RESOURCES_DIR/ssl_keystore.jks" ]; then
+  echo "$RESOURCES_DIR/ssl_keystore.jks does not exist; cannot run test" >&2
+  exit 11
+fi
```

Comment:
When this script is ran, it's on a Docker image that includes all of the resources needed to run the tests against Presto+Kerberos, right? I'm assuming this script isn't meant to work anywhere else besides that Docker image since `RESOURCES_DIR` is hardcoded... why not just make this test script part of the Docker image too? Or part of the GH action config

Reply:
Right now, the GitHub action running this test doesn't use any special Docker image.  It just uses the one provided ultimately by `mba`, which I'm pretty sure comes from the `.devcontainer` one.

In order for the script to be available to the GitHub action, it would need to be moved into `.github`, which I'm not sure is mounted into the Docker container.  I'll give that a try.  I'd rather not introduce another `Dockerfile` definition just to hold one script.

Reply:
I've decided to simply move the script into `.github` to hopefully make it clear that it's only intended to work in that context.  I don't really want to invest a bunch more time pulling everything back out piecemeal into the GitHub action, since that was the initial path I took, and debugging/iteration was an absolute nightmare/time sink.